docs/modules/_result_.html

```diff
@@ -162,7 +162,7 @@ <h3 id="examples-fluent-object-invocation">Examples: fluent object invocation</h
 <span class="hljs-keyword">const</span> okLen = hooray.map(length).unwrapOr(<span class="hljs-number">0</span>); <span class="hljs-comment">// okLen = 3</span>
 
 <span class="hljs-keyword">const</span> muySad = err(<span class="hljs-string">'oh no'</span>);
-<span class="hljs-keyword">const</span> errLen = hooray.map(length).unwrapOr(<span class="hljs-number">0</span>); <span class="hljs-comment">// errLen = 0</span>
+<span class="hljs-keyword">const</span> errLen = muySad.map(length).unwrapOr(<span class="hljs-number">0</span>); <span class="hljs-comment">// errLen = 0</span>
```

Comment:
:wat:

Reply:
Heh. It's a docs fix from @pixelhandler earlier today that didn't get rebuilt in the docs.